sig-scalability/throughput_burts_slo.md

```diff
@@ -0,0 +1,23 @@
+# SLO: Kubernetes cluster of size at least X is able to start Y Pods in Z minutes
+
+## Burst Pod Startup Throughput SLO
+### Summary
+Kubernetes cluster of size at least X is able to start Y Pods in Z minutes.
+### User Stories
+- User is running a workload of X total pods and wants to ensure that it can be started in Y time. 
+- User wants to understand how quickly they can react to a dramatic change in workload profile - e.g. they need to begin scheduling new pods when QPS meets W% of max to accommodate a spike due to a Black Friday Sale.
+- User is running a huge serving app on a huge cluster. He wants to know how quickly he can recreate his whole setup in case of a serious disaster which will bring the whole cluster down.
+
+Current steady state SLOs are do not provide enough data to make these assessments about burst behavior. 
+## SLO definition (full)
+### Test setup
+Standard performance test kubernetes setup.
+### Test scenario is following:
+- Start with a healthy (all nodes ready, all cluster addons already running) cluster with N running pause Pods/Node
```

Comment:
I'd like to know if the value of N is zero or not.  Also, more complicated pods would affect this test; those that have resource limits that require the scheduler to do a little more work in fitting, use different scheduler features, etcetera.